src/ATen/function_wrapper.py

```diff
@@ -276,6 +280,24 @@ def find_first_tensor(formals):
     def format_formal(f):
         return '{} {}'.format(f['type'], f['name'])
 
+    def get_broadcast_argument(option):
+        for argument in option['arguments']:
+            if argument.get('broadcast'):
+                return argument
+
+    def get_broadcast_actuals(broadcast_arg, broadcast_inplace, broadcast_dims):
```

Comment:
This looks codegen-y. Can we add some documentation here?

Reply:
done.